.github/workflows/make-debs.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Build .deb files'
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        base_image: ["ubuntu:bionic", "ubuntu:disco", "ubuntu:eoan", "debian:buster"]
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: docker build with arg
+        run: |
+          mkdir -p output
+          image_name=$(echo ${{ matrix.base_image }} | cut -d: -f2)
```

Comment:
Maybe I'd avoid this kind of logic and use the feature from GitHub Actions:

```yaml
matrix:
  base_image: [...]
  include:
    - base_image: "ubuntu:bionic"
      image_name: "bionic"
    ...
    - base_image: "debian:buster"
      image_name: "buster"
```

then remove this shell variable and use `${{ matrix.image_name }}` instead ([reference][community]).

[community]: https://github.community/t5/GitHub-Actions/GitHub-Actions-Matrix-options-dont-work-as-documented/td-p/29558

Or can we use `${{ matrix.base_image }}` also as the image name maybe?

Reply:
OK, but `include:` is a special directive which append additional pattern to matrix outside of the combination. So use some other name `names:`